### PR TITLE
Added option to enable/disable verbose mode

### PR DESCRIPTION
**Pull request by janimattiellonen**

### CODE REVIEW

**Comment by naxhh** [comment]
Check your IDE, you've broken some identations in the vagrantfile :(

Also you have translated all our cool spaces to tabs, why?
https://travis-ci.org/phansible/phansible/jobs/45793209

Check that.
About the Idea, I'll discuss it in the issue you have created.

**Comment by janimattiellonen** [comment]
Ah, apparently using tabs :( Fixing this right now

**Comment by mikeSimonson** [comment]
@janimatti The indentation is still wrong, you can see it on the diff. It's especially true for the [vagrant_file](https://github.com/phansible/phansible/pull/167/files#diff-1).

**Comment by janimattiellonen** [comment]
I ran the validation tool on my local machine and it didn't complain.

**Comment by naxhh** [comment]
I've downloaded the  branch and the diff in github is being displayed correctly.

You have broken some identations there.

Example:
```
{% if enableWindows %}
    # Check to determine whether we're on a windows or linux/os-x host,
    # later on we use this to launch ansible in the supported way
    # source: https://stackoverflow.com/questions/2108727/which-in-ruby-checking-if-program-exists-in-path-from-ruby
    def which(cmd)
    exts = ENV['PATHEXT'] ? ENV['PATHEXT'].split(';') : ['']
    ENV['PATH'].split(File::PATH_SEPARATOR).each do |path|
    exts.each { |ext|
    exe = File.join(path, "#{cmd}#{ext}")
    return exe if File.executable? exe
    }
    end
    return nil
    end
{% endif %}
```

**Comment by janimattiellonen** [comment]
Ok. Seems like the travis and Scrutinizer validations pass though. This is a bit weird, no?

File: tests/src/Phansible/Renderer/VagrantfileRendererTest.php [hunk 1]
```diff
@@ -35,6 +35,7 @@ public function testLoadDefaults()
         $this->assertEquals('./', $this->model->getSyncedFolder());
         $this->assertEquals('nfs', $this->model->getSyncedType());
         $this->assertEquals(true, $this->model->isWindowsEnabled());
+		$this->assertEquals(true, $this->model->isVerboseModeEnabled());
```
Comment (naxhh) [inline tests/src/Phansible/Renderer/VagrantfileRendererTest.php]: identations :(

**Comment by naxhh** [comment]
Pretty sure phpcs only check php files ;)

**Comment by naxhh** [comment]
I'm rejecting it as don't seems a good idea. Discussed in #166

I like @sndpl proposal and I will accept that PR if you want to do that :)

Sorry for all the confusion of the PR identations and stuff.

**Pull request closed without merging**